docs/index.html

```diff
@@ -1 +1,131 @@
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="contribute.html">Contribute</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="download.html">Download</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    
+    <header class="masthead">
+      <div class="overlay">
+        <div class="container">
+          <h1 class="display-1 text-white">Stay on track.</h1>
+          <h2 class="display-5 text-white">Life gets hard and keeping track of homework and exam dates is not the first thing on your mind. Raider Planner will be there to help
+          you out.</h2>
+        </div>
+      </div>
+    </header>
+    
+        <section>
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-md-6 order-2">
+            <div class="p-5">
+              <img src="images/calendar.png" alt="">
+            </div>
+          </div>
+          <div class="col-md-6 order-1">
+            <div class="p-5">
+              <h2 class="display-4">Get organized with a monthly view calendar...</h2>
+              <p>All tasks and activities will be updated to your calender so you can get a month overview without the hasal of putting in the dates.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-md-6">
+            <div class="p-5">
+              <img src="images/reminder.png" alt="">
+            </div>
+          </div>
+          <div class="col-md-6">
+            <div class="p-5">
+              <h2 class="display-4">Reminders so you remain alert...</h2>
+              <p>You won't be forgetting about that exam next week without an alert popping up to tell you it's time to study!</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-md-6 order-2">
+            <div class="p-5">
+              <img src="images/gantt.png" alt="" style="width:350px; height:300px">
+            </div>
+          </div>
+          <div class="col-md-6 order-1">
+            <div class="p-5">
+              <h2 class="display-4">Keep track of your progress...</h2>
+              <p>Use the Gantt diagram to view how far you have gone and how much more you need to accomplish!</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Footer -->
+    <footer class="py-3 bg-dark">
+      <div class="footer">
+        <p> Would you like to contribute to this project? Click <a href="contribute.html">HERE</a> to find out more information </p>
+        <p class="m-0 text-center text-white">Copyright &copy; Raider Planner 2017</p>

```

Comment:
RaiderPlanner is not an organization, so this should rather say "RaiderPlanner Contributors" and then link to the AUTHORS file in the project repository. I have just now added the file.